.github/workflows/build.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
       - name: Install dependencies
-        run: python3 -m pip install tox
+        run: python3 -m pip install tox --break-system-packages tox
```

Comment:
what's the need for `--break-system-packages` now?

Reply:
I started running into issues after upping from 22.04 to 24.04 on the gh runners:

https://github.com/canonical/container-runner-operator/actions/runs/11068460084/job/30753946445

Reply:
Probably should have been its own pr :grimacing: